backend/handlers/user_institutions_handler.py

```diff
@@ -0,0 +1,44 @@
+# -*- coding: utf-8 -*-
+"""User Institutions Handler."""
+
+from google.appengine.ext import ndb
+import json
+
+from utils import login_required
+from utils import Utils
+from utils import json_response
+from send_email_hierarchy.leave_institution_email_sender import LeaveInstitutionEmailSender
+
+from handlers.base_handler import BaseHandler
+
+
+class UserInstitutionsHandler(BaseHandler):
+    """Handle user's operations relationed to an especific institution."""
```

Comment:
if is specific institution why userInstitutioS?

Reply:
Because actually we're handling with a collection of institutions.